File: language/c++/libraries/stack/classes/stack.cpp
#include <iostream>
#include <stack>
using namespace std;

int main() {
  stack<string> s;

  s.push("red");
  s.push("green");
  s.push("blue");

  // Consume elements from stack
  while (!s.empty()) {
    cout << s.top() << endl;
    s.pop();
  }

  return 0;
}

File: language/gpl/dart/packages/flutter/widgets/classes/RenderObjectWidget/Stack.dart
import 'package:flutter/material.dart';

/**
 * - Position widgets on top of each other
 * - E.g., an image with a text in the foreground
 */

Widget main() {
  return Stack(
    children: <Widget>[
      Container(
        width: 100,
        height: 100,
        color: Colors.red,
      ),
      Container(
        width: 90,
        height: 90,
        color: Colors.green,
      ),
      Container(
        width: 80,
        height: 80,
        color: Colors.blue,
      ),
    ],
  );
}
